packages/twenty-front/src/modules/workflow/workflow-variables/utils/filterOutputSchema.ts

```diff
@@ -60,7 +63,7 @@ const filterRecordOutputSchema = ({
       filteredFields[key] = {
         ...field,
         value: validSubSchema,
-      };
+      } as FieldOutputSchemaV2;
```

Comment:
avoid as! 
Should be fixable properly: 
- if your as is added on top of a partial, this means that you can use Pick<> in the function using it
- if it's because of union type, we should add something to discriminate the union on ts side